src/kernels/variables/debuggerVariables.node.ts

```diff
@@ -316,7 +319,9 @@ export class DebuggerVariables
             // Run our dataframe scripts only once per session because they're slow
             const key = this.debugService.activeDebugSession?.id;
             if (key && !this.importedDataFrameScriptsIntoKernel.has(key)) {
-                await this.evaluate(DataFrameLoading.DataFrameSysImport);
+                const scriptPath = path.join(EXTENSION_ROOT_DIR, DataFrameLoading.ScriptPath);
```

Comment:
Could debuggerVariables also be made isomorphic at this point? At a quick glance it looked like the same usage of scripts as pythonVariableRequester.

Reply:
Checking!